indico/web/forms/fields/principals.py

```diff
@@ -63,6 +67,7 @@ def __init__(self, *args, **kwargs):
         self.allow_emails = kwargs.pop('allow_emails', False)
         self.groups = kwargs.pop('groups', False)
         self.allow_networks = kwargs.pop('allow_networks', False)
+        self.allow_category_roles = kwargs.pop('allow_category_roles', True)
```

Comment:
If you don't need it in the end (due to using PermissionsField in category protection), remove it from here again. 

Otherwise document the new kwarg in the docstring above.

Reply:
I don't think this is resolved - either remove the change to this field (if it's not needed), otherwise add it to the docstring...